src/app/shared/navbar/navbar.component.ts

```diff
@@ -137,12 +137,16 @@ export class NavbarComponent implements OnInit, AfterContentChecked, OnDestroy {
     this.authService.logout();
   }
 
-  //naviagtes to the user information by using the authservice class in the constructor above 
+  /**
+   * naviagtes to the users information by using the 
+   * @constructor authService 
```

Comment:
The `@constructor` tag is used to denote a function that intended to be called with the "new" keyword (see http://usejsdoc.org/tags-class.html). With that tag taken out though, the comment looks good!